SDK docs Spryk section review and edits (#1354)
spryker/spryker-docs: docs/sdk/dev/development-tools/performance-audit-tool-benchmark.md

You can write tests without any additional knowledge, but Benchmark has some default helpers that can make writing benchmarks easier for Spryker:

> * **RequestBuilder** - helps you to build the `Request` object.

Reviewer: ```suggestion
* **RequestBuilder**—helps you to build the `Request` object.
```

> * **HttpHelper** - sends a request.

Reviewer: ```suggestion
* **HttpHelper**—sends a request.
```

> * **LoginHelper** - allows you to log in with some credentials during or before the benchmark.

Reviewer: use "let" instead of "allow" https://developers.google.com/style/word-list#allows-you-to
for em-dashes, don't use spaces
```suggestion
* **LoginHelper**—lets you log in with some credentials during or before the benchmark.
```

> * **FormCsrfTokenHelper** - allows you to get valid CSRF token if you want to submit a form on the page.

Reviewer: ```suggestion
* **FormCsrfTokenHelper**—lets you get a valid CSRF token if you want to submit a form on the page.
```
